### ISSUE TITLE
Timeout waiting for ACK/NACK after 'Get ChipID (0x28)'

### ISSUE THREAD

**Issue by hocomadvies** [JelmerT/cc2538-bsl#125]
`pi@raspberrypi:~ $ sudo python3 cc2538-bsl/cc2538-bsl.py /dev/ttyUSB0 -evw  --bootloader-sonoff-usb  CC2652RB_coordinator_20220219.hex
`
```
Opening port /dev/ttyUSB0, baud 500000
Connecting to target...
ERROR: Timeout waiting for ACK/NACK after 'Get ChipID (0x28)'
```

Tried to update the slaesh cc2652RB stick through ssh on my rpi3b. I kept on getting the Timeout waiting for ACK/NACK. Took out the stick and replaced it. Made sure the port is ok. Read trough all posts/ how-to''s about this subject but still cannot get it to work. After applying some install / update I now get this:

```
pi@raspberrypi:~ $ sudo python3 cc2538-bsl/cc2538-bsl.py /dev/ttyUSB0 -evw  --bootloader-sonoff-usb  CC2652RB_coordinator_20220219.hex
Opening port /dev/ttyUSB0, baud 500000
Connecting to target...
CC1350 PG2.0 (7x7mm): 352KB Flash, 20KB SRAM, CCFG.BL_CONFIG at 0x00057FD8
Primary IEEE Address: 00:12:4B:00:21:CC:39:4A
```

Does this mean it has been flashed? Went so fast ? How can I check if its ok?

**Comment by hocomadvies** [comment]
tried also with pushing the button when inserting. Nothing different.

**Comment by hocomadvies** [comment]
Still getting errors when trying to flash. it seems not working ok. How to deal with it get it working ok?

**Comment by JelmerT** [comment]
It seems the chip on your stick can be read, but the script quit too early?

Are you still having issues?

**Comment by nielsnl68** [comment]
Yes, i do have the same issue, just brought this sonoff ZBDongle-P 3.0 dongle as well.
And wanted to upgrade my firmware. with your phyton script.
